.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*]
+indent_style=space
+indent_size=tab
+tab_width=4
```

Comment:
I don't believe that editorconfig is worth having in a repo when the code is always formatted with cargo-fmt (and this is checked on travis-ci)

Reply:
I think having .editorconfig may help users that use Visual Studio code, since new lines of code are formatted according to that config. See https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2019

Reply:
@ozkriff both, my editor's (vim) and my cargo-fmt default settings are different to the one used in this repo. This file really makes life easier if you're switching between multiple projects and you don't remember/don't care what kind of formatting they use